pkgs/top-level/all-packages.nix

```diff
@@ -15748,6 +15749,13 @@ in
 
   intel-ocl = callPackage ../os-specific/linux/intel-ocl { };
 
+  intel-aesmd = callPackage ../os-specific/linux/intel-aesmd/2.7.0.nix { };
+  intel-aesmd2_3_1 = callPackage ../os-specific/linux/intel-aesmd/2.3.1.nix { };
+  intel-aesmd2_4_0 = callPackage ../os-specific/linux/intel-aesmd/2.4.0.nix { };
+  intel-aesmd2_5_0 = callPackage ../os-specific/linux/intel-aesmd/2.5.0.nix { };
+  intel-aesmd2_6_0 = callPackage ../os-specific/linux/intel-aesmd/2.6.0.nix { };
+  intel-aesmd2_7_0 = callPackage ../os-specific/linux/intel-aesmd/2.7.0.nix { };
```

Comment:
Why the many different versions?

Reply:
The protocol has changed through these versions, and the latest release of https://crates.io/crates/aesm-client which interfaces with aesmd only works up to 2.4.0. This is why I added these alternative versions and not just the latest one.

Reply:
Than I only would add 2.4 and the latest.